electron/renderer/src/selector/AccountSelector.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
```

Comment:
```suggestion
 * Copyright (C) 2023 Wire Swiss GmbH
```

Reply:
dammit beat me on the only cr I can do :D